tests/subsys/dsp/basicmath/testcase.yaml

```diff
@@ -1,6 +1,6 @@
 tests:
   libraries.cmsis_dsp.basicmath:
```

Comment:
These are no longer CMSIS-DSP tests, so the test names need to be updated as well:

```suggestion
  dsp.basicmath:
```

Reply:
Done